package.json

```diff
@@ -33,7 +33,7 @@
     "openseadragon": "https://github.com/TheScienceMuseum/openseadragon/tarball/smg",
     "page": "^1.7.1",
     "rc": "^1.1.6",
-    "slug": "^0.9.1",
+    "slugg": "^1.1.0",
```

Comment:
good explanation of reason for changing module: https://github.com/TheScienceMuseum/collectionsonline/issues/775#issuecomment-275661971 (_thanks_!)